docs/posta-servisleri.md

```diff
@@ -1,3 +1,14 @@
 # E-posta Servisleri
 
-> Eklenecek
+> ### ProtonMail
+>Ücretsiz ve ücretli planlar, uçtan uca şifreleme, sıfır erişim şifrelemesi, kimlik avına karşı koruma, spam önleme ve 2FA dahil tüm güvenlik özelliklerini içerir. Kendi Android, iOS Uygulamaları ve web arayüzü. Ücretli planlar özel alan adlarını etkinleştirir. Tor Ağı üzerinden erişilebilir.
```

Comment:
\> kullanmaya gerek yok. Paragraflar, ara notları göstermek için kullanılmalı. [Lütfen VPN sayfasını bir örnek olarak inceleyin!](https://guvendekal.org/#/vpn)

Hizmetin özellikleri belirtilmeli (açık kaynak mı? fiyatı ne kadar (aylık baz alınmalı)? şirket hangi ülkede?)

Resim kullanımı öneririm.

Reply:
İlk başta yazıları eklemeyi sonrasında görselleri eklemeyi düşündüm hızlı olması açısından tavsiyelerinizi dikkate alıp tekrardan hazırlayacağım teşekkürler